CHANGELOG.md

```diff
@@ -1,4 +1,10 @@
-1.13.0 / TBC
+1.14.0 / TBC
+===================
+- Use full path for Open Graph image
+- Do not use capitalised convention for constant variables
```

Comment:
Any reasoning for this? ALL CAPS gives a visual indication that that variable should be set somewhere

Reply:
Sure, however, this is more of a hangover from previous versions of JS that didn't have the `const` keyword.